It seems we still have issues with the multicluster workflow: https://github.com/cilium/cilium/actions/runs/1001641767
```
📋 Test Report
❌ 2/9 tests failed (12/89 actions), 0 warnings, 0 tests skipped, 2 scenarios skipped:
Test [no-policies]:
  ❌ no-policies/pod-to-remote-nodeport/curl-0: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-other-node (echo-other-node:8080)
  ❌ no-policies/pod-to-remote-nodeport/curl-4: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
  ❌ no-policies/pod-to-remote-nodeport/curl-5: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
  ❌ no-policies/pod-to-remote-nodeport/curl-6: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-other-node (echo-other-node:8080)
  ❌ no-policies/pod-to-remote-nodeport/curl-10: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
  ❌ no-policies/pod-to-remote-nodeport/curl-11: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
Test [allow-all]:
  ❌ allow-all/pod-to-remote-nodeport/curl-2: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-other-node (echo-other-node:8080)
  ❌ allow-all/pod-to-remote-nodeport/curl-4: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
Error: Connectivity test failed: 2 tests failed
  ❌ allow-all/pod-to-remote-nodeport/curl-5: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
  ❌ allow-all/pod-to-remote-nodeport/curl-8: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-other-node (echo-other-node:8080)
  ❌ allow-all/pod-to-remote-nodeport/curl-10: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
  ❌ allow-all/pod-to-remote-nodeport/curl-11: cilium-test/client2-666976c95b-blprg (10.76.1.228) -> cilium-test/echo-same-node (echo-same-node:8080)
Error: Process completed with exit code 1.
```
However, as @tklauser was saying offline:
> At least it's only 2/9 tests failed compared to 9/9 tests failed before :smile:

Soooo I guess it's progress...?

--- a/.github/workflows/conformance-aks.yaml
+++ b/.github/workflows/conformance-aks.yaml
@@ -248,6 +248,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Run connectivity test
         run: |
@@ -279,6 +280,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Restart connectivity test pods
         run: |

--- a/.github/workflows/conformance-aks.yaml
+++ b/.github/workflows/conformance-aks.yaml
@@ -246,7 +246,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Run connectivity test
@@ -256,7 +256,7 @@ jobs:
       - name: Clean up Cilium
         run: |
           cilium uninstall --wait
-          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+          pkill -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay"
 
       - name: Create custom IPsec secret
         run: |
@@ -277,7 +277,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Restart connectivity test pods

--- a/.github/workflows/conformance-aws-cni.yaml
+++ b/.github/workflows/conformance-aws-cni.yaml
@@ -245,6 +245,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Run connectivity test
         run: |

--- a/.github/workflows/conformance-aws-cni.yaml
+++ b/.github/workflows/conformance-aws-cni.yaml
@@ -243,7 +243,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Run connectivity test

--- a/.github/workflows/conformance-eks.yaml
+++ b/.github/workflows/conformance-eks.yaml
@@ -223,6 +223,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Run connectivity test
         run: |
@@ -254,6 +255,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Restart connectivity test pods
         run: |

--- a/.github/workflows/conformance-eks.yaml
+++ b/.github/workflows/conformance-eks.yaml
@@ -221,7 +221,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Run connectivity test
@@ -231,7 +231,7 @@ jobs:
       - name: Clean up Cilium
         run: |
           cilium uninstall --wait
-          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+          pkill -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay"
 
       - name: Create custom IPsec secret
         run: |
@@ -252,7 +252,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Restart connectivity test pods

--- a/.github/workflows/conformance-gke.yaml
+++ b/.github/workflows/conformance-gke.yaml
@@ -207,6 +207,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Run connectivity test
         run: |
@@ -234,6 +235,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Restart connectivity test pods
         run: |
@@ -270,6 +272,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Restart connectivity test pods
         run: |
@@ -304,6 +307,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Restart connectivity test pods
         run: |

--- a/.github/workflows/conformance-gke.yaml
+++ b/.github/workflows/conformance-gke.yaml
@@ -205,7 +205,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Run connectivity test
@@ -215,7 +215,7 @@ jobs:
       - name: Clean up Cilium
         run: |
           cilium uninstall --wait
-          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+          pkill -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay"
 
       - name: Install Cilium with tunnel datapath
         run: |
@@ -232,7 +232,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Restart connectivity test pods
@@ -247,7 +247,7 @@ jobs:
       - name: Clean up Cilium
         run: |
           cilium uninstall --wait
-          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+          pkill -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay"
 
       - name: Create custom IPsec secret
         run: |
@@ -268,7 +268,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Restart connectivity test pods
@@ -284,7 +284,7 @@ jobs:
       - name: Clean up Cilium
         run: |
           cilium uninstall --wait
-          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+          pkill -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay"
 
       - name: Install Cilium with encryption and tunnel datapath
         run: |
@@ -302,7 +302,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Restart connectivity test pods

--- a/.github/workflows/conformance-kind.yaml
+++ b/.github/workflows/conformance-kind.yaml
@@ -89,6 +89,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Run connectivity test
         run: |
@@ -116,6 +117,7 @@ jobs:
         run: |
           cilium hubble port-forward&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Restart connectivity test pods
         run: |

--- a/.github/workflows/conformance-kind.yaml
+++ b/.github/workflows/conformance-kind.yaml
@@ -87,7 +87,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Run connectivity test
@@ -97,7 +97,7 @@ jobs:
       - name: Clean up Cilium
         run: |
           cilium uninstall --wait
-          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+          pkill -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay"
 
       - name: Install Cilium with encryption
         run: |
@@ -114,7 +114,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
           sleep 10s
 
       - name: Restart connectivity test pods

--- a/.github/workflows/conformance-multicluster.yaml
+++ b/.github/workflows/conformance-multicluster.yaml
@@ -260,9 +260,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward \
-            --context ${{ steps.contexts.outputs.context1 }} \
-            -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward --context ${{ steps.contexts.outputs.context1 }}&
           sleep 10s
 
       - name: Run connectivity test

--- a/.github/workflows/conformance-multicluster.yaml
+++ b/.github/workflows/conformance-multicluster.yaml
@@ -284,8 +284,7 @@ jobs:
           cilium connectivity test \
             --context ${{ steps.contexts.outputs.context1 }} \
             --multi-cluster ${{ steps.contexts.outputs.context2 }} \
-            --test '!pod-to-nodeport' \
-            --test '!pod-to-local-nodeport' \
+            --test '!/pod-to-.*-nodeport' \
              --flow-validation=disabled
 
       - name: Post-test information gathering

--- a/.github/workflows/conformance-multicluster.yaml
+++ b/.github/workflows/conformance-multicluster.yaml
@@ -262,6 +262,7 @@ jobs:
         run: |
           cilium hubble port-forward --context ${{ steps.contexts.outputs.context1 }}&
           sleep 10s
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
 
       - name: Run connectivity test
         run: |

